bin/auth/insert_tokens.py

```diff
@@ -1,12 +1,31 @@
 import argparse
 import emission.storage.decorations.token_queries as esdt
+import emission.core.get_database as edb
 
 if __name__ == '__main__':
     parser = argparse.ArgumentParser(prog="insert_tokens")
 
-    parser.add_argument("file")
+    parser.add_argument("--file", help="add tokens to DB from file")
+    parser.add_argument("--single", help="add a single token")
+    parser.add_argument("--uuid", help="add tokens to the DB off of the email field of UUIDS", action="store_true")
+    parser.add_argument("--show", help="show tokens that are in the DB", action="store_true")
```

Comment:
what happens if I call this with no arguments?
what happens if I call it with multiple arguments?

You should add unit tests for those cases as well.

Hint: Have you considered using a mutually exclusive group?